internals/overlord/checkstate/handlers.go

```diff
@@ -184,3 +184,9 @@ func pluralise(n int, singular, plural string) string {
 	}
 	return fmt.Sprintf("%d %s", n, plural)
 }
+
+func (m *CheckManager) RunCheck(ctx context.Context, check *plan.Check) error {
+	logger.Debugf("Performing check %q", check.Name)
```

Comment:
Probably don't need this log as the POST /v1/checks/run will already be recorded, and each checker prints a debug log too.